src/openrct2/windows/ride.c

```diff
@@ -4010,8 +4010,25 @@ static void window_ride_maintenance_paint(rct_window *w, rct_drawpixelinfo *dpi)
 	// Mechanic status
 	if (ride->lifecycle_flags & RIDE_LIFECYCLE_BROKEN_DOWN) {
 		switch (ride->mechanic_status) {
-		case RIDE_MECHANIC_STATUS_CALLING:
-			stringId = STR_CALLING_MECHANIC;
+		case RIDE_MECHANIC_STATUS_CALLING:{
+			sint32 count_staff = peep_get_staff_count();
+			rct_peep *peep;
+			uint16 spriteIndex;
+			sint32 count = 0;
+			for(sint32 i = 0; i < count_staff; ++i){
+				FOR_ALL_STAFF(spriteIndex, peep){
+					if(peep->staff_type == STAFF_TYPE_MECHANIC){
+						++count;
```

Comment:
Please change this to `count++;`, this is more in line with the rest of the codebase.